src/core/core_structs/simulations_stages.jl

```diff
@@ -60,6 +60,9 @@ end
 mutable struct Stage <: AbstractStage
     op_problem::Type{<:AbstractOperationsProblem}
     model::OperationsTemplate
+    initial_time::Dates.DateTime
+    horizon::Int64
```

Comment:
This information will be duplicated in canonical. I am fine with that, just warning about it.